content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/scrap_inventory.rst

```diff
@@ -2,93 +2,122 @@
-   :guilabel:`Storage Locations` under the :guilabel:`Warehouse` heading.
+.. tip::
+   To view the all-time total quantities of scrapped items, navigate to :menuselection:`Inventory
+   app --> Configuration --> Locations`. Click the :guilabel:`x (remove)` button on the
+   :guilabel:`Internal` filter in the :guilabel:`Search...` bar, to display virtual locations.
+
+   Select the :guilabel:`Virtual Locations/Scrap` location. From the :guilabel:`Scrap` location's
+   form, click the :guilabel:`Current Stock` smart button, at the top of the form.
+
+   A list of all scrapped products and their quantities is displayed.
+
+   .. image:: scrap_inventory/scrap-inventory-current-stock.png
+      :align: center
+      :alt: Current Stock list of all scrapped products in virtual scrap location.
 
-Alternatively, a list of all delivery orders, receipts, and transfers can be viewed by navigating to
-:menuselection:`Inventory --> Operations --> Transfers`.
+Scrap from an existing operation
+================================
 
-Next, open a delivery order, receipt, or transfer from the corresponding page by clicking on it. A
-:guilabel:`Scrap` button appears at the top of the page. Click it to open the :guilabel:`Scrap`
-pop-up window.
+Scrap orders (SPs) can *also* be created from existing operations, such as receipts, delivery
+orders, and internal transfers, before they are entered into or removed from stock for an operation.
 
-.. image:: scrap_inventory/scrap-pop-up.png
+To scrap a product during an operation, navigate to the :menuselection:`Inventory app`. From the
+:guilabel:`Inventory Overview`, locate the :guilabel:`# To Process` button on an operation's task
+card (i.e. the :guilabel:`Receipts` task card).
+
+.. image:: scrap_inventory/scrap-inventory-receipts-task-card.png
    :align: center
-   :alt: The scrap pop-up in the Inventory app.
+   :alt: # To Process button on Receipts task card on Inventory Overview page.
+
+Click the :guilabel:`# To Process` button, and select an operation to process from the resulting
+list of existing orders. Doing so opens that operation's form.
 
-.. important::
-   The :guilabel:`Scrap` button will only appear on a receipt that has been validated. This is
-   because Odoo only allows products to be scrapped once they have been entered into inventory.
+Click the :guilabel:`⚙️ Actions (gear)` icon, and select :guilabel:`Scrap` from the list of
+resulting options. This opens a :guilabel:`Scrap Products` pop-up window.
+
+.. image:: scrap_inventory/scrap-inventory-popup-window.png
+   :align: center
+   :alt: Scrap Products pop-up window on operation form.
 
-On the :guilabel:`Scrap` pop-up window, select the product being scrapped from the
-:guilabel:`Product` drop-down menu. Then, enter the quantity in the :guilabel:`Quantity` field.
+From this pop-up, click the drop-down in the :guilabel:`Product` field, and select the products from
```

Comment:
```suggestion
From this pop-up window, click the drop-down menu in the :guilabel:`Product` field, and select the products from
```

Reply:
do you think it might be beneficial to mention that a 'Lot/Serial' field might appear after a 'Product' is selected?

Reply:
Nice catch - did my testing with a non-tracked product, so didn't think of this! Suggestion implemented ✅